### ISSUE TITLE
Clarify hardware requirements in the installation guide

### ISSUE THREAD

**Issue by AyaMorisawa** [misskey-dev/misskey#4983]
> @syuilo I think this information should be documented to the installation guide.

_Originally posted by @acid-chicken in https://github.com/syuilo/misskey/issues/4826#issuecomment-488282853_

**Comment by u1-liquid** [comment]
The memory specification problem is actually due to the peak at the time of WebPack build, 
Therefore, there are likely to be many differences in memory specifications between what is required for actual operation and what is required for build.
I can say if server have 2GB of RAM, it will be enough to build and running both Misskey and Mastodon instance with running PostgreSQL, Redis, Ai-Chan on same machine.

**Comment by sftblw** [comment]
IMHO, with fairy enough swap for webpack build, 1GB is fine or minimum requirement to run misskey.
my instance runs misskey (install), elasticsearch for misskey, gitea (docker - barely used), nginx on 1GB arch linux VPS.

Previously I set swap file as 2GB but that was not enough. I think 3~4GB swap would be  appropriate. I experienced "Can't Build" when It was v10 and after that set up [automated swap allocation](https://wiki.archlinux.org/index.php/Swap#Automated), and the problem fixed.
